src/hal/src/device.rs

```diff
@@ -265,10 +265,16 @@ pub trait Device<B: Backend>: Any + Send + Sync {
     unsafe fn destroy_pipeline_layout(&self, layout: B::PipelineLayout);
 
     /// Create a pipeline cache object.
-    unsafe fn create_pipeline_cache(&self, data: Option<&[u8]>) -> Result<B::PipelineCache, OutOfMemory>;
+    unsafe fn create_pipeline_cache(
```

Comment:
note: if you are applying rustfmt changes, please make it a separate commit independent of the functional changes

Reply:
Reverted them, didnt notice this when I ran it